fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/digitalcash/DigitalCashReceiveFragment.java

```diff
@@ -110,4 +115,18 @@ public void handleOnBackPressed() {
               }
             });
   }
+
+  /**
+   * Gets the color of the QR code based on the current night mode configuration.
+   *
+   * @return the color of the QR code (either Color.WHITE or Color.BLACK)
+   */
+  public int getQRCodeColor() {
+    Configuration configuration = getResources().getConfiguration();
+    int nightModeFlags = configuration.uiMode & Configuration.UI_MODE_NIGHT_MASK;
+    if (nightModeFlags == Configuration.UI_MODE_NIGHT_YES) {
+      return Color.WHITE;
+    }
+    return Color.BLACK;
+  }
```

Comment:
To avoid code replication I was wondering if it would be better to have such function written only once as static, perhaps taking as parameter a `Configuration`, so that it has not to be replicated multiple times across the different fragments.

Reply:
Yes you are right, I was thinking about this but couldn't really find a spot where it fits... Do you have a suggestion?

Reply:
And about the text contrast, good point, I will change that!

Reply:
Probably it could fit as utility function in ActivityUtils, but there might be other better places